Installation failed for both node version 4 and 5

4 warnings and 10 errors generated.
make: **\* [Release/obj.target/binding/src/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/jpnath/.nvm/versions/node/v4.3.1/lib/node_modules/vue-dashing-js/node_modules/node-sass/node_modules/pangyp/lib/build.js:272:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 15.3.0
gyp ERR! command "/Users/jpnath/.nvm/versions/node/v4.3.1/bin/node" "/Users/jpnath/.nvm/versions/node/v4.3.1/lib/node_modules/vue-dashing-js/node_modules/node-sass/node_modules/pangyp/bin/node-gyp" "rebuild" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /Users/jpnath/.nvm/versions/node/v4.3.1/lib/node_modules/vue-dashing-js/node_modules/node-sass
gyp ERR! node -v v4.3.1
gyp ERR! pangyp -v v2.3.3
gyp ERR! not ok 
Build failed
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/jpnath/.nvm/versions/node/v4.3.1/bin/node" "/Users/jpnath/.nvm/versions/node/v4.3.1/bin/npm" "install" "-g" "vue-dashing-js"
npm ERR! node v4.3.1
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE

npm ERR! node-sass@3.1.2 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@3.1.2 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jpnath/Develop/Production/npm-debug.log

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
         "fs-extra": "~0.18.4",
         "jade": "~0.35.0",
         "mincer": "~1.2.4",
-        "node-sass": "~3.3.3",
+        "node-sass": "~4.5.0",
         "node-schedule": "~0.2.8",
         "prompt": "~0.2.14",
         "request": "~2.57.0",

